src/js/init.js

```diff
@@ -0,0 +1 @@
+console.log("Hi there!")
```

Comment:
Missing blank line at the end of file, FYI

Reply:
Thank you for noticing.